src/terminal/terminal.rs

```diff
@@ -197,22 +201,28 @@ where
     ///
     /// Requested size will be saved so the size can remain consistent when rendering. This leads
```

Comment:
Should this docstring be changed to "Requested _area_ will be saved"

Reply:
good catch!
e9215f0422f69c8003d2a6252d190f0f882497e6